### ISSUE TITLE
New submissions via control panel not connected to current user

### ISSUE THREAD

**Issue by michaelhue** [verbb/formie#613]
**Description**

New form submissions created via the control panel are not connected to the current user.
Maybe this is intended behavior? For my use case it would be very helpful to have the user saved as well.

**Steps to reproduce**
1. Create new form submission
2. Check user in submission index

**Additional info**
- Plugin version: 1.4.22
- Craft version: Craft Pro 3.7.17.2
- Multi-site: No

**Comment by engram-design** [comment]
Added for the next release, also with the ability to change the user. To get this early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.4.22",
  "...": "..."
}
```

Then run `composer update`.

**Comment by engram-design** [comment]
Fixed in [1.4.23](https://github.com/verbb/formie/releases/tag/1.4.23)
